### ISSUE TITLE
JavaScript output is disabled in JupyterLab app

### ISSUE THREAD

**Issue by djinnome** [jupyterlab/jupyterlab_app#242]
Hi folks

## Description
![Screen Shot 2021-09-24 at 1 40 09 PM](https://user-images.githubusercontent.com/144210/134737169-1955e8fd-22f2-46c5-a62c-eefefa2f8e89.png)

## Reproduce

1. Start JupyterLab app
2. In any code cell, type `%matplotlib inline` or any other command that uses javascript.
3.  See warning: `JavaScript output is disabled in JupyterLab`

## Expected behavior

Javascript works in notebook and regular Jupyter lab.

## Context

- Operating System and version: Mac OS X Mojave
- Browser and version: Chrome Version 93.0.4577.82 (Official Build) (x86_64)
- JupyterLab version: Version 3.1.12-1

**Comment by djinnome** [comment]
I am still getting the same error message in JupyterLab version: Version 3.1.12-2

Do I need to install JupyterLab app from source to get this change, or am I just doing something wrong?

**Comment by mbektas** [comment]
@djinnome we haven't published a new version with the fix yet, please wait for it (~1-2 days). otherwise you can build from master by following the instructions in [README.md](https://github.com/jupyterlab/jupyterlab_app/blob/master/README.md).

**Comment by djinnome** [comment]
Confirmed that the fix works in Version 3.1.13-1 (3.1.13-1) 

However, when I tried to copy the version number in the "About jupyterlab_app" window, I got a pop-up window with the following error message:

**A JavaScript error occurred in the main process**

```
Uncaught Exception:
TypeError: Cannot read property 'copy' of null at KeyboardShortcutManager.copy 
(/Applications/JupyterLab.app/Contents/Resources/app.asar/build/out/main/shortcuts.js:57:55)
```
